app/services/work_ocr_creator_remover.rb

```diff
@@ -0,0 +1,39 @@
+# Given a work:
+#   figures out which assets need OCR;
+#   arranges for their OCR to be created (either via a job or immediately) or removed,
+#   depending on the work's `ocr_requested`.
+#
+#   Ignores child works completely.
+class WorkOcrCreatorRemover
+  attr_reader :work
+
+  # @param work [Work] Work to look at
+  # can be used to update a progress UI.
+  def initialize(work, ignore_missing_files=false)
```

Comment:
can you make this a keyword argument, instead of mysterious positional argument?

compare (current, worse) `WorkOcrCreatorRemover.new(work, true)` to (better) `WorkOcrCreatorRemover.new(work, ignore_missing_files: true)`. 

Thanks!

Reply:
Sure - makes sense.